scp/css/scp.css

```diff
@@ -344,7 +344,7 @@ a.emailSettings { background:url(../images/icons/emails.png) }
 a.emailDiagnostic { background:url(../images/icons/email_diagnostic.gif) }
 a.banList { background:url(../images/icons/ban_list.gif) }
 
-a.users { background:url(../images/icons/list_users.gif) }
+a.users { background:url(../images/icons/user.gif) }
```

Comment:
The list_users.gif icon ![](https://raw.githubusercontent.com/antriver/osTicket-1.8/feature/assigned-icons/scp/images/icons/list_users.gif) shows a person with a "ticket" in front of them. That's appropriate when it's shown next to the "My Tickets" link [![](http://img.ctrlv.in/img/15/11/16/564a54b730702.png)](http://ctrlv.in/669494), but the same icon was also used for the "Agents" and "Access" pages in the admin panel. Since I added that new icon (user.gif ![](https://raw.githubusercontent.com/antriver/osTicket-1.8/feature/assigned-icons/scp/images/icons/user.gif)) to the assigned to column, it seemed more appropriate to use that one when referring to agents as individual people. [![](http://img.ctrlv.in/img/15/11/16/564a54db4796b.png)](http://ctrlv.in/669495)